Mathlib/Computability/PartrecCode.lean

```diff
@@ -211,14 +211,14 @@ theorem encode_lt_pair (cf cg) :
 
 theorem encode_lt_comp (cf cg) :
     encode cf < encode (comp cf cg) ∧ encode cg < encode (comp cf cg) := by
-  suffices; exact (encode_lt_pair cf cg).imp (fun h => lt_trans h this) fun h => lt_trans h this
```

Comment:
The literal translation would be
```lean
suffices ?_ by
  exact (encode_lt_pair cf cg).imp (fun h => lt_trans h this) fun h => lt_trans h this
```
right? I don't mind you removing @digama0's golf here, but it's reassuring to know that these gymnastics remain possible with the new syntax.